### DISCUSSION TITLE
Set collection inheritance

### DISCUSSION THREAD

**Discussion by CAPITAINMARVEL** [BeanieODM/beanie#698] [Question]
I can’t figure out how i can find a way to add a specific collection name on inheritance it seems to be the class name

**Comment by mrrobinhood5** [comment]
@CAPITAINMARVEL from https://beanie-odm.dev/tutorial/defining-a-document/ you will set up a `Settings` class with `name = 'collection_name`. I you want to make this a parent class, follow https://beanie-odm.dev/tutorial/inheritance/ , you need to set the `is_root = True` inside the `Settings` class. Inherited documents will use the same collection as its parent.

**Reply by CAPITAINMARVEL** [reply]
its what i did try i got a collection named Items so i renamed my class like 
```python
class Item(Document):
    name: str
    class Settings:
        name = "Items"
        is_root = True

class Mount(Item):
```
Then i did a print on a find and the list was empty and when i removed the name "Items" and put back Items as class name it was querying a document

**Reply by mrrobinhood5** [reply]
So then you would find child documents like 
```py
item = await Mount.find({}).to_list()

# or using the parent
item = await Item.find({}, with_children=True).to_list()
```

Of course, I'm assuming you initiated beanie prior to this
